vllm/config/speculative.py

```diff
@@ -31,7 +31,7 @@
 
 SpeculativeMethod = Literal["ngram", "eagle", "eagle3", "medusa",
                             "mlp_speculator", "draft_model", "deepseek_mtp",
```

Comment:
not directly related to this pr, but do we need a different mtp method for every model.

Reply:
Yeah this is the case currently as it seems to be needed to dispatch to *_mtp.py for each model